roles/aodh/templates/etc/aodh/aodh.conf

```diff
@@ -5,6 +5,8 @@ rpc_backend = rabbit
 log_dir = /var/log/aodh
 debug = {{ aodh.debug }}
 
+rest_notifier_ssl_verify = False
```

Comment:
for the python-memcached, it's needed for aodh and ceilometer.  
it's not included in this change set. you can add it.